deeprank_gnn/ginet.py

```diff
@@ -1,43 +1,33 @@
 import torch
-from torch.nn import Parameter
 import torch.nn.functional as F
-import torch.nn as nn
+from torch import nn
 
 from torch_scatter import scatter_mean
 from torch_scatter import scatter_sum
 
-from torch_geometric.utils import remove_self_loops, add_self_loops, softmax
-
 # torch_geometric import
 from torch_geometric.nn.inits import uniform
 from torch_geometric.nn import max_pool_x
-from torch_geometric.data import DataLoader
 
 # deeprank_gnn import
 from deeprank_gnn.community_pooling import get_preloaded_cluster, community_pooling
-from deeprank_gnn.NeuralNet import NeuralNet
-from deeprank_gnn.DataSet import HDF5DataSet, PreCluster
 
 
 class GINetConvLayer(torch.nn.Module):
+    def __init__(self, in_channels, out_channels, number_edge_features=1, bias=False):
```

Comment:
same as above, here `number_edge_features` argument is not used, but if I remove it the tests break (even if the argument doesn't seem used by the tests). Any idea about why does this happen?